features/filter_recipes.feature

```diff
@@ -42,11 +42,17 @@ Scenario: API call returns 3 dinner recipes
 Scenario: API call returns no breakfast recipes
   When API call returns no breakfast recipes
   Then I should see nothing in breakfast
+  And I should see 3 recipes in lunch
+  And I should see 3 recipes in dinner
 
 Scenario: API call returns no lunch recipes
   When API call returns no lunch recipes
   Then I should see nothing in lunch
+  And I should see 3 recipes in breakfast
+  And I should see 3 recipes in dinner
 
 Scenario: API call returns no dinner recipes
   When API call returns no dinner recipes
-  Then I should see nothing in dinner
\ No newline at end of file
+  Then I should see nothing in dinner
+  And I should see 3 recipes in breakfast
```

Comment:
I'm glad you added these in - it makes the tests more specific and comprehensive! Your changes look fine to me

Reply:
Good idea putting sad path